src/components/Inngest.ts

```diff
@@ -89,6 +90,8 @@ export class Inngest<TOpts extends ClientOptions = ClientOptions> {
    */
   public readonly inngestBaseUrl: URL;
 
+  public readonly inngestAPI: InngestAPI;
```

Comment:
In the spirit of hiding this from users when they type `inngest.` in their editor, can we make this `private` and refer to it using brackets elsewhere? e.g. `inngest["inngestApi"]` instead of `inngest.inngestApi`